src/less/overwrites/_forms.less

```diff
@@ -49,13 +51,13 @@ textarea{
   }
 }
 
-
 .input-group-addon{
   background-color: #e1e6ec;
 }
 
-label.control-label.required::after{
-  content: '*';
+label.control-label.required{
   font-weight: bold;
-
+  &:after{
+    content: ' *';
```

Comment:
According to dieter, yes 

https://jira.hybris.com/browse/CCE-269?focusedCommentId=387325&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-387325